recipes/ligo-common/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: ligo-common
+  version: {{version}}
+
+source:
+  fn: ligo-common-{{version}}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/ligo-common-{{version}}.tar.gz
```

Comment:
For readability, would encourage using `{{ version }}` with spaces between `{`/`}` and text.